LICENSE.md

```diff
@@ -1,33 +1,49 @@
-The MIT License (MIT)
-=====================
+MXE is licensed under the MIT Licence. The makefiles and utilities under
+`plugins/`, `src/`, and `tools/` all share this licence.
 
-Copyright (c) 2007-2015
+> Copyright (c) 2007-2016
+>
+>   * Volker Diels-Grabsch
+>   * Mark Brand
+>   * Tony Theodore
+>   * Martin Gerhardy
+>   * Tiancheng "Timothy" Gu
+>   * Boris Nagaev
+>   * ... and many other [contributors][contributors]
+>
+> (contact via the [project mailing list][mailing-list] or
+> [issue tracker][issue-tracker])
+>
+> Permission is hereby granted, free of charge, to any person obtaining a copy
+> of this software and associated documentation files (the "Software"), to deal
+> in the Software without restriction, including without limitation the rights
+> to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+> copies of the Software, and to permit persons to whom the Software is
+> furnished to do so, subject to the following conditions:
+>
+> The above copyright notice and this permission notice shall be included in all
+> copies or substantial portions of the Software.
+>
+> THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+> IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+> FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+> AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+> LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+> OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+> SOFTWARE.
 
-  * Volker Diels-Grabsch
-  * Mark Brand
-  * Tony Theodore
-  * Martin Gerhardy
-  * Tiancheng "Timothy" Gu
-  * Boris Nagaev
-  * ... and many other contributors
+MXE distributes code from the following projects in the `ext/` directory:
 
-(contact via the project mailing list)
+- [GNU config.guess][config.guess] [GPL3+ with Autoconf exception]
+- [GNU Make Standard Library][gmsl] [BSD 3-clause]
```

Comment:
Could you provide links to texts of "GPL3+ with Autoconf exception" and "BSD 3-clause", please?